src/translations/ru/index.js

```diff
@@ -1,11 +1,13 @@
 import app from './app';
 import dashboard from './dashboard';
 import randomProductsBanner from './randomProductsBanner';
+import happyWizardForm from './happyWizardForm';
 
 const combinedTranslations = {
   ...app,
   ...dashboard,
   ...randomProductsBanner,
+  ...happyWizardForm,
```

Comment:
I think it should be a general `...form,` with all form translation. Or at least all general things should be moved there.